libs/wingsdk/src/target-sim/schema-resources.ts

```diff
@@ -72,11 +74,26 @@ export interface QueueSchema extends BaseResourceSchema {
   };
 }
 
+export interface EventSubscription {}
```

Comment:
(a little bit above)

I think we can remove `subscribers` from `QueueSchema.props` since we're tracking all of the subscribers using the event subscriptions (this will be a breaking change but I think it's appropriate here) / they're no longer need to be provided by the Queue's `toSimulator()` method